### DISCUSSION TITLE
AttributeError: module '_pythonTestCPP' has no attribute 'delete_testPy'

### DISCUSSION THREAD

**Discussion by hkrishna-git** [swig/swig#2212] [Q&A] [answered]
Hi Team,

I was generating the SWIG interfaces for python.
I was able to generate the interfaces successfully & able to build. I followed SWIG documentation for generation procedure. 
But while importing the module, I get the following error :
>>> import pythonTestCPP
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\hsagala\source\repos\python\pythonTestCPP.py", line 347, in <module>
    class testPy(object):
  File "C:\Users\hsagala\source\repos\python\pythonTestCPP.py", line 353, in testPy
    __swig_destroy__ = _pythonTestCPP.delete_testPy
AttributeError: module '_pythonTestCPP' has no attribute 'delete_testPy'

Could someone help me in resolving this issue ? I have attached the relevant documents over here in the zip file.

![import_err_2022-02-14-215903](https://user-images.githubusercontent.com/54533871/153905045-485a7bae-8515-4832-9793-c1b0a8684bee.jpg)
[pythonTestCPP.zip](https://github.com/swig/swig/files/8062400/pythonTestCPP.zip)

Thank you very much in advance.

**Comment by ojwb** [comment, marked as answer]
I don't know what's going wrong, but it sounds like a toolchain setup or usage issue so if you're still stuck I'd suggest trying to get a really simple example working first, e.g. `Examples/python/simple/` in the SWIG source tree.

If you need more help, the swig-user mailing list is probably a better place to ask.  Enabling github discussions was a bit of an experiment, but it doesn't seem to be working as a way to get help because very few people seem to read them so you're lucky if someone who can answer sees your question.
